packages/react/src/components/Panel/index.tsx

```diff
@@ -1,33 +1,41 @@
-import React from 'react';
+import React, { HTMLAttributes, ReactElement, ReactNode } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import IconButton from '../IconButton';
 
-const Panel = ({
-  className,
-  title = '',
-  actions,
-  children
-}: {
+interface PanelProps extends HTMLAttributes<HTMLElement> {
+  children: ReactNode;
+  heading:
```

Comment:
I did this to match what we are doing in the Dialog component in order to encourage good semantics and allow customization: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Dialog/index.tsx#L19-L24